tests/publiccloud/patch_and_reboot.pm

```diff
@@ -30,7 +30,7 @@ sub run {
 
     ssh_fully_patch_system($remote);
 
-    $args->{my_instance}->softreboot(timeout => get_var('PUBLIC_CLOUD_REBOOT_TIMEOUT', 600));
+    $args->{my_instance}->softreboot(timeout => get_var('PUBLIC_CLOUD_REBOOT_TIMEOUT', 1200));
```

Comment:
```suggestion
    $args->{my_instance}->softreboot(timeout => get_var('PUBLIC_CLOUD_REBOOT_TIMEOUT', (get_var('PUBLIC_CLOUD_SLES4SAP') ? 1200 : 600));
```

Reply:
not even that , instead of tweaking default value we can just define a variable in certain case . this is the whole point of having variable here :) otherwise we can just drop variable and simply always tweak value in place :)

Reply:
I will withdraw this PR and using PUBLIC_CLOUD_REBOOT_TIMEOUT instead.
Thanks [asmorodskyi](https://github.com/asmorodskyi) your comment makes sense